sympy/solvers/tests/test_solvers.py

```diff
@@ -386,7 +386,16 @@ def test_solve_transcendental():
 
     eq = 2*(3*x + 4)**5 - 6*7**(3*x + 9)
     result = solve(eq, x)
-    ans = [(log(2401) + 5*LambertW(-log(7**(7*3**Rational(1, 5)/5))))/(3*log(7))/-1]
+    ans = S("""
+        [-(log(2401) + 5*LambertW(-(-1 + sqrt(5) + sqrt(2)*I*sqrt(sqrt(5) +
+        5))*log(7**(7*3**(1/5)/20))))/(3*log(7)),
+        -(log(2401) + 5*LambertW((1 + sqrt(5) - sqrt(2)*I*sqrt(5 -
+        sqrt(5)))*log(7**(7*3**(1/5)/20))))/(3*log(7)),
+        -(log(2401) + 5*LambertW((1 + sqrt(5) + sqrt(2)*I*sqrt(5 -
+        sqrt(5)))*log(7**(7*3**(1/5)/20))))/(3*log(7)),
+        -(log(2401) + 5*LambertW((-sqrt(5) + 1 + sqrt(2)*I*sqrt(sqrt(5) +
+        5))*log(7**(7*3**(1/5)/20))))/(3*log(7)),
+        -(log(2401) + 5*LambertW(-log(7**(7*3**(1/5)/5))))/(3*log(7))]""")
```

Comment:
I forgot how to extract minus sign here. It is failing because of that although the result is correct. checked from [Wolfram](https://www.wolframalpha.com/input/?i=solve+2*(3*x+%2B+4)**5+-+6*7**(3*x+%2B+9)+for+x)

Reply:
Do mean how to get the minus sign not to distribute as `(-2-x)/y` on smthng like `-(2+x)/y`? If so, put the -1 last: `(2+x)/y*-1`....or perhaps just with the denominator as `(2+x)/-y`. You just have to keep it from appearing just before an Add into which it is automatically distributed.

Reply:
```python
>>> (2+sqrt(2))/(-log(7))
-(sqrt(2) + 2)/log(7)
```